src/machine/machined-dbus.c

```diff
@@ -863,7 +863,7 @@ static int method_set_pool_limit(sd_bus_message *message, void *userdata, sd_bus
                 return 1; /* Will call us back */
 
         /* Set up the machine directory if necessary */
-        r = setup_machine_directory(error);
+        r = setup_machine_directory(error, true, true);
```

Comment:
Why this takes two `true`?

Reply:
This is called from `SetPoolLimit`, thus we assume intent on user's part. Also, this retains the existing behavior of automatically enabling quotas because systemd-machined behavior is not the focus of this PR.

Reply:
Ah, indeed.